src/js/common/domutil.js

```diff
@@ -594,7 +595,20 @@ var userSelectProperty = domutil.testProp([
 ]);
 var supportSelectStart = 'onselectstart' in document;
 var prevSelectStyle = '';
+
 /* eslint-enable*/
+/**
+ * If the target is not a popup, it prevents the default.
+ * @method
+ * @param {MouseEvent} event - Mouse event object
+ */
+domutil.preventDefaultWhenNotPopup = function(event) {
```

Comment:
This method and name is not extensible. If you have another element to disable text selection, you will must change this method and name.